### PR TITLE
Add all the fonts found by fuzzers to the repo

### PR DESCRIPTION
**Pull request by ebraminio**
Also listed them on test/shaping/data/in-house/tests/fuzzed.tests

### CODE REVIEW

**Comment by behdad** [comment]
Please move them to test/fuzzing/fonts

File: test/shaping/data/in-house/tests/fuzzed.tests [hunk 1]
```diff
@@ -21,3 +21,87 @@
...
+../../../../fuzzing/fonts/clusterfuzz-testcase-minimized-hb-fuzzer-6210176798425088:--font-funcs=ot:U+0041:*
+../../../../fuzzing/fonts/clusterfuzz-testcase-minimized-hb-fuzzer-6260579246276608:--font-funcs=ot:U+0041:*
+../../../../fuzzing/fonts/clusterfuzz-testcase-minimized-hb-fuzzer-6264625609834496:--font-funcs=ot:U+0041:*
+../../../../fuzzing/fonts/clusterfuzz-testcase-minimized-hb-fuzzer-6424351550210048:--font-funcs=ot:U+0041:*
+../../../../fuzzing/fonts/clusterfuzz-testcase-minimized-hb-fuzzer-6460279560863744:--font-funcs=ot:U+0041:*
+../../../../fuzzing/fonts/clusterfuzz-testcase-minimized-hb-fuzzer-6576177596596224:--font-funcs=ot:U+0041:*
+../../../../fuzzing/fonts/clusterfuzz-testcase-minimized-hb-fuzzer-6595199411159040:--font-funcs=ot:U+0041:*
+../../../../fuzzing/fonts/clusterfuzz-testcase-minimized-hb-fuzzer-6624904746106880:--font-funcs=ot:U+0041:*
+../../../../fuzzing/fonts/clusterfuzz-testcase-minimized-hb-fuzzer-6723367514144768:--font-funcs=ot:U+0041:*
+#../../../../fuzzing/fonts/clusterfuzz-testcase-minimized-hb-shape-fuzzer-5630246225707008:--font-funcs=ot:U+0041:*
```
Comment (ebraminio) [inline test/shaping/data/in-house/tests/fuzzed.tests]: Note this please

**Comment by ebraminio** [comment]
Also uploaded https://github.com/google/oss-fuzz/pull/1879

**Comment by ebraminio** [comment]
Rushed merge of this as google/oss-fuzz#1879 merge

**Pull request merged**